### ISSUE TITLE
 'diagnose' is an invalid keyword argument for exceptions logging

### ISSUE THREAD

**Issue by jonathanhle** [Delgan/loguru#101]
I'm getting the following trying to add a default logger:
`logger.add("/var/log/protectwise_bootstrap.log", backtrace=True, diagnose=True, format="{time:YYYY-MM-DD at HH:mm:ss} | {level} | {message}")
`

> TypeError: 'diagnose' is an invalid keyword argument for this function

For the most part, I think I'm just copying the examples in the doc.

**Comment by ms7m** [comment]
Use the [Stable](https://loguru.readthedocs.io/en/stable/index.html) docs. Pretty sure the docs on the master isn't the same version as on pip.

**Comment by Delgan** [comment]
Yeah, this is indeed an issue with the Readme version being ahead compared to the latest published release on PyPI. I learned to wait before updating the Readme next time. :grimacing: 

 The `diagnose` option will be available in the next v0.3.0 release to control whether or not the variables values should be displayed in the exception stacktrace.

I removed the `diagnose` part in the Readme and will re-add it once next version is published (hopefully soon).
